UserAgent/Home View/HomeViewController.swift

```diff
@@ -68,7 +128,7 @@ class HomeViewController: UIViewController {
 
     private lazy var historyView: HistoryView = {
         let historyView = HistoryView(profile: self.profile)
-        historyView.delegate = self
+//        historyView.delegate = self
```

Comment:
If we don't need to listen HistoryView delegate, please delete it istead of comment.